### ISSUE TITLE
button's function call not working.  

### ISSUE THREAD

**Issue by CodeThePeace** [chinchang/web-maker#375]
## Meta info

- Seems to be a bug:
- Version (3.6.2): The version is no longer displayed in the help section.
- Context - Chrome extension:

Greetings,

>
```
<button onclick="CallFunction()">Click me</button>
```

Does not seem call the function.  

The CSS panel recognizes the button and can be styled and reacts like an active button is programed to do except for implementing the HTML portion of the code.

Entering the code without a function call does implement the code. (See example below.)  

Other variations such as:

> 
```
<button onclick="Alert()">Click me </button>
```
or

> 
```
<button onclick="counter=counter + 1">Click me </button>
```

similarly, do not run the requested code.

**Comment by chinchang** [comment]
@CodeThePeace would it be possible to copy/paste the error message in the developer tool console when you get this error? It seems like something is wrong with your installation. Does refreshing fix the problem? If the problem only shows sometimes, is there a way I can replicate it on my side?

**Comment by CodeThePeace** [comment]
Regrettably, there is no error message.  The function simply does not get called.  This is a moot point since the console does not open on a double click. 

![Screenshot 2019-03-10 at 8 38 49 PM](https://user-images.githubusercontent.com/48228155/54099060-aa10b600-4374-11e9-8f11-f8f5c6c21c1b.png)

**Comment by CodeThePeace** [comment]
Console tool does open now on doubleclick, but there are no messages, errors or otherwise in the console tool.

**Comment by CodeThePeace** [comment]
Pleased to report, this issue was resolved in the newer version ( 4.0 )

**Comment by chinchang** [comment]
As you have already noticed, your code will work inside the Web app. But not in Chrome extension due to security restrictions in browser extensions. In chrome extension, in the HTML panes you can write JavaScript or use event attributes like `onclick` etc.
